wagtailcontact/forms.py

```diff
@@ -0,0 +1,51 @@
+from django import forms
+from django.conf import settings
+from django.core.mail import EmailMessage
+from django.template import loader
+
+
+class ContactForm(forms.Form):
+
+    subject_template = "wagtailcontact/email/subject.txt"
+    txt_template = "wagtailcontact/email/message.txt"
+    success_url = None
+
+    name = forms.CharField(max_length=100, required=True)
+    email = forms.EmailField(max_length=200, required=True)
```

Comment:
`EmailField` comes with a default `max_length`, don't think you need to specify your own.